### ISSUE TITLE
Rotation attemps via fbcon fail

### ISSUE THREAD

**Issue by RJVB** [damentz/liquorix-package#53]
Setting console rotation via

```
$ echo 1 >  /sys/class/graphics/fbcon/rotate_all
```

fail with the liquorix kernels I tried. This is annoying on hybrid tablet PCs that require rotation in order to be used in the common landscape orientation (= with the keyboard attached).

**Comment by damentz** [comment]
@RJVB I found a configuration in Liquorix that didn't match other common distribution settings that might be related.  A new kernel will be published soon with the update.  Can you report back if setting `rotate_all` works with the new kernel?

**Comment by RJVB** [comment]
>@RJVB I found a configuration in Liquorix that didn't match other common distribution settings that might be related.  A new kernel will be published soon with the update.  Can you report back if setting `rotate_all` works with the new kernel?

Sure - if I notice the update and it's a 5.9 kernel (as said elsewhere I can't update beyond that version for now).

But you can tell me what setting you suspect and what you think it should be set to; I also have my own kernel builds which do support `rotate_all` so I could check in the corresponding kconfig.

**Comment by damentz** [comment]
This is the patch of the commit linked to this issue:

```
commit 1377c603d662087ba97c46748275bf6279d1d25f
Author: Steven Barrett <steven@liquorix.net>
Date:   Thu Feb 11 16:27:29 2021 -0600

    config: Reduce CONFIG_FBDEV_OVERALLOC from 300 to 100
    
    As reported in #53, setting `rotate_all` for `fbcon` fails with Liquorix
    but not other kernels.  Investigating configuration differences,
    Liquorix uses a 300% overallocation for FBDEV to emulate triple
    buffering.  However, in both Arch and Debian's latest configurations,
    this value is set to 100.
    
    Use the more common value so Liquorix behaves more similarly to common
    stock kernels.

diff --git a/linux-liquorix/debian/config/kernelarch-x86/config-arch-64 b/linux-liquorix/debian/config/kernelarch-x86/config-arch-64
index ef08f38..e2f8119 100644
--- a/linux-liquorix/debian/config/kernelarch-x86/config-arch-64
+++ b/linux-liquorix/debian/config/kernelarch-x86/config-arch-64
@@ -6207,7 +6207,7 @@ CONFIG_DRM_DP_AUX_CHARDEV=y
 CONFIG_DRM_KMS_HELPER=m
 CONFIG_DRM_KMS_FB_HELPER=y
 CONFIG_DRM_FBDEV_EMULATION=y
-CONFIG_DRM_FBDEV_OVERALLOC=300
+CONFIG_DRM_FBDEV_OVERALLOC=100
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_DP_CEC=y
 CONFIG_DRM_TTM=m
```

And I don't know if this will resolve your issue, but it's pretty clear it's not a common configuration.  Maybe triple buffering and rotated consoles don't mix well?

**Comment by damentz** [comment]
And regarding being stuck on 5.9, Phoronix put out an article January 6th reporting the first version of ZFS supporting 5.10 (2.0.1): https://www.phoronix.com/scan.php?page=news_item&px=OpenZFS-2.0.1-Released

Looking online, Debian's packaging 2.0.2 in unstable: https://packages.debian.org/sid/zfs-dkms.  Not sure what distro you're running but there's options out there.

**Comment by damentz** [comment]
Marked as closed for now since I'm not revisiting the 5.9 branch.  If you can reproduce in 5.10 I'll reopen and we can troubleshoot more.

**Comment by RJVB** [comment]
>Marked as closed for now since I'm not revisiting the 5.9 branch.  If you can reproduce in 5.10 I'll reopen and we can troubleshoot more.

I could try to build the package from source with the patch applied, but kernel builds take hours on my hardware... and annoying as it is, this issue isn't exactly something I run into every day...
